ReactAndroid/src/main/java/com/facebook/react/modules/network/NetworkingModule.java

```diff
@@ -76,43 +76,58 @@
   /* package */ NetworkingModule(
       ReactApplicationContext reactContext,
       @Nullable String defaultUserAgent,
-      OkHttpClient client,
+      OkHttpClientProvider okHttpClientProvider,
       @Nullable List<NetworkInterceptorCreator> networkInterceptorCreators) {
     super(reactContext);
 
+    mClient = createOkHttpClient(okHttpClientProvider, networkInterceptorCreators);
+    mCookieHandler = new ForwardingCookieHandler(reactContext);
+    mCookieJarContainer = (CookieJarContainer) mClient.cookieJar();
+    mShuttingDown = false;
+    mDefaultUserAgent = defaultUserAgent;
+    mRequestIds = new HashSet<>();
+  }
+
+  private OkHttpClient createOkHttpClient(
+    OkHttpClientProvider okHttpClientProvider,
```

Comment:
You can add the  `@Nullable` annotation just like `List<NetworkInterceptorCreator>`